test/ngrok.postinstall.spec.js

```diff
@@ -0,0 +1,111 @@
+const homedir = require('homedir');
+const path = require('path');
+const forge = require('node-forge').pki;
+const fs = require('fs');
+const child_process = require('child_process');
+
+const postinstallPath = path.join(__dirname, '../postinstall.js');
+const ngrokPath = path.join(homedir(), '/.ngrok');
+const certpath = `${ngrokPath}/testCert.pem`;
+const npmrcPath = path.join(homedir(), '.npmrc');
+const npmrcBackupPath = path.join(homedir(), '.npmrc.bak');
+
+describe.only('postinstall cert', () => {
```

Comment:
great test! don't forget to remove "only" =)
you can also spy the request.get and test if CA was actually passed

Reply:
Alright, after a bit more research (as I said, security isn't my strong suit), it looks to me like we won't be able to generate a CA as it'll need to be a valid root certificate on the build agent. If you want to install a cert authority there, I can use it as the npmrc path, but this will not work on local machines that are running the test suite.

Anything I'm misunderstanding?

Reply:
What if we just introduce NGROK_ROOT_CA env var, add it to readme and it will be up to consumers to specify it or not ?

Reply:
I'm fine with that option as well, but the logic will be about the same. We'll have to read the environment variable, read the file, and parse the certs in the file. This would address your comment below, though, and I think that's reasonable. I'll change it up.